sycl/test/check_device_code/matrix-nvptx-double-test.cpp

```diff
@@ -0,0 +1,99 @@
+// RUN: %clangxx -fsycl-device-only -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend --cuda-gpu-arch=sm_80 -DSYCL_EXT_ONEAPI_MATRIX=3 -S -Xclang -emit-llvm %s -o -| FileCheck %s
```

Comment:
I thought I already put a comment about this:
this test should be with the other matrix tests under sycl/test/matrix, right?

Reply:
Please see this [comment](https://github.com/intel/llvm/pull/4696#discussion_r730902699) for the answer to your previous comment.
There is also an [integration test](https://github.com/intel/llvm-test-suite/compare/intel...JackAKirk:matrix-double?expand=1) that I will open a PR in the test-suite for as soon as the current PR is merged.